app/application/services/auth/signup_service.py

```
@@ -0,0 +1,33 @@
+            password=request.password,
+            name=request.name,
+            phonenumber=request.phonenumber,
+            college=request.college,
+            department=request.department,
+            major=request.major,
+        )
+
+        try:
+            await self._auth_repository.add_user(user=user)
+        except DatabaseIntegrityException:
+            raise
```

Ruff (0.12.2)

Remove exception handler; error is immediately re-raised

(TRY203)